cmd/operator/main.go

```diff
@@ -282,20 +290,27 @@ func Main() int {
 		return 1
 	}
 
-	err = cc.CheckPrerequisites(ctx,
-		namespaces(cfg.Namespaces.AllowList).asSlice(),
-		verbs,
-		monitoringv1alpha1.SchemeGroupVersion.String(),
-		monitoringv1alpha1.PrometheusAgentName)
+	scrapeConfigSupported, err := checkPrerequisites(ctx, logger, cc, allowedNamespaces, verbs, monitoringv1alpha1.SchemeGroupVersion.String(), monitoringv1alpha1.ScrapeConfigName)
```

Comment:
we need different verbs for the ScrapeConfig CRD:

```suggestion
	scrapeConfigSupported, err := checkPrerequisites(
		ctx,
		logger,
		cc,
		allowedNamespaces,
		map[string][]string{
			monitoringv1alpha1.ScrapeConfigName: {"get", "list", "watch"},
		},
		monitoringv1alpha1.SchemeGroupVersion.String(),
		monitoringv1alpha1.ScrapeConfigName,
	)

```

Reply:
Good eye, don't know how I overlooked this.